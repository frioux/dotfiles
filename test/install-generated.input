#!/bin/bash

dir='test/,install-generated'
makefile='test/,install-generated.Makefile'
file='test/,install-generated.foo'

cat >"$makefile" <<END
TARGETS_GENERATED := $file

$file:
	echo $$ >\$@

include Makefile
END

make --always-make -f "$makefile" >/dev/null \
  "INSTALLATION_DIR=$dir" \
  'TARGETS_STATIC=Makefile README' \
  install

find "$dir"

rm -rf "$dir" "$makefile" "$file"

# __END__
