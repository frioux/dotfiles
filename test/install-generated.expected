test/,install-generated
test/,install-generated/Makefile
test/,install-generated/README
test/,install-generated/test
test/,install-generated/test/,install-generated.foo
